### PR TITLE
June '24 Update OTP-UI

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
- Support latest OTP (Geocoder/location field)
- More query support (core-utils)
- 

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2397,10 +2397,10 @@
   resolved "https://registry.yarnpkg.com/@opentripplanner/building-blocks/-/building-blocks-1.0.3.tgz#74de32daba0d7fae7d10d5738db6d2f2824da3dd"
   integrity sha512-065pZZTzR+qisgbvJd9swhitMMin1/ooMwuzqqRknsKYK+wOGge3Y3sbgQZATyjR/oH43uF0P2ZhGcbgqgM+YQ==
 
-"@opentripplanner/core-utils@^11.2.3", "@opentripplanner/core-utils@^11.3.1", "@opentripplanner/core-utils@^11.4.0":
-  version "11.4.0"
-  resolved "https://registry.yarnpkg.com/@opentripplanner/core-utils/-/core-utils-11.4.0.tgz#f8c664d67d36108eab5e0477966c156ccfac6619"
-  integrity sha512-4weAgo3PwqrDfaQnLVhGxX67/lpjXQsW6OoIFluFFK6HZEN6mu6KW6OkTBGRbTSlVRE+Xz0FwmPuSrI3WlQDMA==
+"@opentripplanner/core-utils@^11.2.3", "@opentripplanner/core-utils@^11.3.1", "@opentripplanner/core-utils@^11.4.1":
```
Comment (amy-corson-ibigroup) [inline yarn.lock:2400]: Do we need three of these or can we deduplicate this?

**Review by amy-corson-ibigroup — APPROVED** [review]
Looking good!

File: yarn.lock (hunk 1 above)
Comment (miles-grant-ibigroup) [reply yarn.lock:2400]: This is de-duped!

**Review by josh-willis-arcadis — APPROVED** [review]
LGTM

**Pull request merged**